### DISCUSSION TITLE
Running chat service using websocket with hugginface/gpt2 model

### DISCUSSION THREAD

**Discussion by jh-skaion** [facebookresearch/ParlAI#3944] [Q&A]
hello team,
i am trying to tun the huggingface/gpt2 model in a chat service environment, i have fine-tune the model with some of the parlai's task.
this is my config file:
![image](https://user-images.githubusercontent.com/82824999/129587833-19dfeb68-f422-4c9d-b463-bb71eb1ba6c9.png)

can you give me some tips or guidance on how to use the huggingface/gpt2 as chat service?

**Comment by jh-skaion** [comment]
Disregard this... i was able to run it. thanks

**Reply by stephenroller** [reply]
Do you mind posting what changed for any future wanderers?
